### PR TITLE
Fix/login page layout

### PR DESCRIPTION
**Pull request by alexanderleegs**
## Problem

This PR makes style changes to our existing login page to make it scale better for large screens.
[figma](https://www.figma.com/file/owgl8SFMKQd2jrCJMVrukR?node-id=19:2&mode=design#595070225)

Before:
Screenshot 2023-10-25 at 5 10 31 PM

After:
Screenshot 2023-10-25 at 5 10 09 PM

### CODE REVIEW

File: src/layouts/Login/LoginPage.tsx [hunk 1]
```diff
@@ -30,7 +30,12 @@ import { useGetSgidAuth, useLogin, useVerifyOtp } from "hooks/loginHooks"
 import { getAxiosErrorMessage } from "utils/axios"
 import { useErrorToast, useSuccessToast } from "utils/toasts"
 
-import { IsomerLogo, LoginImage, OGPLogo, SingpassLogo } from "assets"
```
Comment (kishore03109) [inline src/layouts/Login/LoginPage.tsx:33]: if logos are not longer used, lets remove them!

File: src/layouts/Login/LoginPage.tsx [hunk 2]
```diff
@@ -233,51 +240,43 @@ export const LoginPage = (): JSX.Element => (
...
-          justifyContent="center"
-          pt="5vh"
-        >
+        <Flex h="100%" alignItems="start" justifyContent="center" pt="5vh">
           <LoginContent />
         </Flex>
       </GridItem>
       {/* Custom colour to match stairs */}
-      <GridItem area="credits" bgColor="#E9F0FB" borderColor="#E9F0FB">
+      <GridItem area="credits" bgColor="primary.500" borderColor="primary.500">
```
Comment (kishore03109) [inline src/layouts/Login/LoginPage.tsx:268]: uhm... does we have colour pallets for this ah? if have please use them instead

**Review by kishore03109 — APPROVED** [review]
w nits

File: src/layouts/Login/LoginPage.tsx (hunk 2 above)
Comment (alexanderleegs) [reply src/layouts/Login/LoginPage.tsx:274]: Replaced the whites with pallettes, primary.500 is our palette here i think! https://github.com/isomerpages/isomercms-frontend/pull/1619/commits/34effe40ac540c5a6d506143c4e139d3830644a8

File: src/layouts/Login/LoginPage.tsx (hunk 1 above)
Comment (alexanderleegs) [reply src/layouts/Login/LoginPage.tsx:33]: Removed OGPLogo in https://github.com/isomerpages/isomercms-frontend/pull/1619/commits/b29727ffa1d0655498c19b77918d97d316cf4d6b!

**Pull request merged**